src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1416,7 +1417,7 @@ class ApplicationManagerImpl
   // Thread that pumps messages audio pass thru to mobile.
   impl::AudioPassThruQueue audio_pass_thru_messages_;
 
-  HMICapabilities hmi_capabilities_;
+  std::auto_ptr<HMICapabilities> hmi_capabilities_;
```

Comment:
From the previous closed pull request I have a general question to whom it may concern. In the code there is no similar to C++11 unique_ptr functionality and I wold like to know why. I think that a wrapper would be easier to be updated than every single instance of auto_ptr.